0-1: first task with sum was done

JuliaKirschenheuter opened the pull request.

JuliaKirschenheuter committed:
0-1: first task with sum was done

JuliaKirschenheuter committed:
Merge branch 'master' of https://github.com/js-tasks-ru/vue-20200615_ju-ki

JuliaKirschenheuter committed:
1-1: task with count button was done

JuliaKirschenheuter committed:
Merge branch 'master' of https://github.com/js-tasks-ru/vue-20200615_ju-ki

jsru commented:
Поздравляю, все задачи решены абсолютно верно
```diff
A+B
+ должна быть определена
+ должна получать 1+1=2
+ должна получать 10+20=30
+ должна получать 0+0=0
+ должна получать -2+(-1)=-3
```
```diff
CounterButton
+ Задача добавлена, но требует ручной проверки
```
```diff
Meetup Page Draft
+ Задача добавлена, но требует ручной проверки
```

Добавляю преподавателя (@ShGKme) для код-ревью, посмотрим что он скажет :)

ShGKme commented on 1-module/1-task/script.js:
```diff
@@ -1,4 +1,15 @@
+
+  methods: {
+    onclick($event) {
+      this.buttonNumber = ++ $event.target.value;
```
`$event.target.value` у кнопки изначально отсутствует и здесь не нужен.
Нужно лишь увеличивать значение счётчика `this.buttonNumber`.

ShGKme reviewed (changes requested):
Требуется исправление в задаче `CounterButton` и добавить решение задачи `MeetupsPageDraft`

jsru commented:
Решение было обновлено, посмотрим что скажет @ShGKme

JuliaKirschenheuter committed:
1-2: Meetup Page Draft

localisation data and image cover was done

JuliaKirschenheuter committed:
1-2: Meetup Page Draft

Object.assign was corrected

JuliaKirschenheuter committed:
1-2: Meetup Page Draft

agenda was partly implemented

JuliaKirschenheuter committed:
1-2: Meetup Page Draft

extended version of processedMeetup: not ready state

JuliaKirschenheuter committed:
1-2: Meetup Page Draft

agenda: defaultTitle and icon were done

jsru commented:
Решение было обновлено, посмотрим что скажет @ShGKme

JuliaKirschenheuter committed:
1-2: Meetup Page Draft

code was corrected: not needed methods were removed and start value of meetup was corrected

JuliaKirschenheuter committed:
1-1: task with count button was corrected

JuliaKirschenheuter committed:
Merge branch 'master' of https://github.com/js-tasks-ru/vue-20200615_ju-ki

jsru commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

```diff
Компонент CounterButton
- Компонент должен рендерить кнопку с текстом count
- Компонент должен иметь count 0 по умолчанию
- Компонент должен увеличивать значение на 1 по клику
- Пропс count должен иметь числовой тип
```
```diff
A+B
+ должна быть определена
+ должна получать 1+1=2
+ должна получать 10+20=30
+ должна получать 0+0=0
+ должна получать -2+(-1)=-3
```
```diff
CounterButton
+ Задача добавлена, но требует ручной проверки
```
```diff
Meetup Page Draft
+ Задача добавлена, но требует ручной проверки
```

jsru commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

```diff
Компонент CounterButton
- Компонент должен рендерить кнопку с текстом count
- Компонент должен иметь count 0 по умолчанию
- Компонент должен увеличивать значение на 1 по клику
- Пропс count должен иметь числовой тип
```
```diff
A+B
+ должна быть определена
+ должна получать 1+1=2
+ должна получать 10+20=30
+ должна получать 0+0=0
+ должна получать -2+(-1)=-3
```
```diff
CounterButton
+ Задача добавлена, но требует ручной проверки
```
```diff
Meetup Page Draft
+ Задача добавлена, но требует ручной проверки
```

ShGKme reviewed (approved):
Все задачи приняты 👍

jsru commented:
PR не может быть принят, обратитесь, пожалуйста, к преподавателю.

ShGKme commented:
Не заметил, что добавилась новая задача. Нужно решить все добавленные задачи, чтобы PR был принят

JuliaKirschenheuter committed:
2-1: task with count button was solved with event and prop

JuliaKirschenheuter committed:
2-1: task with count button was solved with v-model

jsru commented:
Решение было обновлено, посмотрим что скажет @ShGKme

ShGKme commented on 2-module/1-task/index.html:
```diff
@@ -9,7 +9,7 @@
   <p><counter-button v-model="count1"></counter-button> - count = {{ count1 }}</p>
   <p><counter-button v-model="count2"></counter-button> - count = {{ count2 }}</p>
-  <p><counter-button :count="count3" @increment="count3 = $event"></counter-button> - count = {{ count3 }}</p>
+  <p><counter-button v-model="count3"></counter-button> - count = {{ count3 }}</p>
```
В файлах вне файлов для редактирования в задаче обычно находятся либо нужные для работы задачи модули, либо примеры для ручного тестирования. Их обычно не нужно изменять, если этого не требуется для тестирования других случаев

ShGKme reviewed (approved):
Принято 👍
